homeassistant/components/media_player/const.py

```diff
@@ -14,6 +14,7 @@
 ATTR_MEDIA_ENQUEUE = "enqueue"
 ATTR_MEDIA_EXTRA = "extra"
 ATTR_MEDIA_EPISODE = "media_episode"
+ATTR_MEDIA_GENRE = "media_genre"
```

Comment:
Please don't change our entity model in this PR. Changing our entity model requires approval in an issue in our architecture repository.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Will do, thanks the clarification!

Reply:
I created this:

https://github.com/home-assistant/architecture/issues/475

Will remove genre from the PR for now until that request has been processed.